docs/source/upcoming_release_notes/736-sxr_solid_attenuators.rst

```diff
@@ -0,0 +1,43 @@
+736 sxr_solid_attenuators
+#########################
```

Comment:
I just realized how these pre-release pages make PRs much easier to review. I can open this page on my second monitor and refer to it as I leaf through the code.